Cargo.toml

```diff
@@ -21,6 +21,9 @@ exclude = [
 	"tests/",
 ]
 
+[package.metadata.cargo-udeps.ignore]
+normal = ["minidom_writer", "md5"]
+
```

Comment:
we really want to use this?

Reply:
I do agree with @datanel, we do not want/cannot update these dependencies yet but it might be possible at some point in the future and putting this section in `Cargo.toml` creates 2 problems for me:
- we acknowledge the use of yet another tool `cargo-udeps` (even if it's a useful tool, no need to make the project dependent on it)
- these dependencies might never be updated anymore (except if we remember to check these conditions each time)

Reply:
There are 2 points here I think.
1. Little misunderstanding: this "ignore" is used to ignore warnings about "unused dependency", not about "not up-to-date".
2. Anyway, I also think we do not want to enforce the use of udeps (my idea was to check it every now and then, manually). So yes, we might as well remove those lines and just wait for tests to crash telling us what's a false-positive of udeps.

TODO if you agree: remove this cargo section.